PWGDQ/Tasks/dqFlow.cxx

```diff
@@ -0,0 +1,396 @@
+    corrconfigs.push_back(fGFW->GetCorrelatorConfig("refP {3} refN {-3}", "ChGap32", kFALSE));
+  }
+
+  // Fill the FlowContainer
+  void FillFC(const GFW::CorrConfig& corrconf, const double& cent, const double& rndm, bool fillflag, bool dqflag)
+  {
+    // Calculate the correlations from the GFW
+    //    double dnx, dny, valx, valy;
+    //    dnx = fGFW->Calculate(corrconf, 0, kTRUE).Re();
+    //    dny = fGFW->Calculate(corrconf, 0, kTRUE).Im();
+    //    if (dnx == 0) {
+    //      return;
+    //    }
+    //
+    //    if (!corrconf.pTDif) {
+    //      valx = fGFW->Calculate(corrconf, 0, kFALSE).Re() / dnx;
+    //      if (TMath::Abs(valx) < 1) {
+    //        fFC->FillProfile(corrconf.Head.Data(), cent, valx, 1, rndm);
+    //        if (dny == 0) {
+    //          return;
+    //        }
+    //        valy = fGFW->Calculate(corrconf, 0, kFALSE).Re() / dny;
+    //      }
+    //      return;
+    //    }
+    //    bool DisableOverlap = kFALSE;
+    //    int nAxisPtBins = 31;
+    //    for (int i = 1; i <= nAxisPtBins; i++) {
+    //      dnx = fGFW->Calculate(corrconf, 0, kTRUE, DisableOverlap).Re();
+    //      if (dnx == 0) {
+    //        return;
+    //      }
+    //      valx = fGFW->Calculate(corrconf, 0, kFALSE, DisableOverlap).Re() / dnx;
+    //      if (TMath::Abs(valx) < 1) {
+    //         fFC->FillProfile(Form("%s_pt_%i", corrconf.Head.Data(), i), cent, valx, 1., rndm);
+    //      }
+    //      return;
+    //    }
+  }
+
+  // Templated function instantianed for all of the process functions
+  template <uint32_t TEventFillMap, uint32_t TTrackFillMap, typename TEvent, typename TTracks>
+  void runFillQvector(TEvent const& collision, aod::BCs const& bcs, TTracks const& tracks1)
+  {
+    // Reset the fValues and Qn vector array
+    VarManager::ResetValues(0, VarManager::kNVars);
+    // Fill the event information with the VarManager
+    VarManager::FillEvent<TEventFillMap>(collision);
+
+    uint64_t tag = 0;
+    // store the selection decisions
+    for (int i = 0; i < kNsel; i++) {
+      if (collision.selection()[i] > 0) {
+        tag |= (uint64_t(1) << i);
+      }
+    }
+    if (collision.sel7()) {
+      tag |= (uint64_t(1) << kNsel); //! SEL7 stored at position kNsel in the tag bit map
+    }
+    int mult = tracks1.size();
+
+    // TODO: properly access to config files from ccdb using bc.timestamp()
+    // auto bc = collision.bc_as<aod::BCsWithTimestamps>();
+
+    if (fcfgAcceptance.value.empty() == false) {
+      // cfg.mAcceptance = ccdb->getForTimeStamp<GFWWeights>(fcfgAcceptance.value, bc.timestamp());
+      if (cfg.mAcceptance) {
+        LOGF(info, "Loaded acceptance histogram from %s (%p)", fcfgAcceptance.value.c_str(), (void*)cfg.mAcceptance);
+      } else {
+        LOGF(warning, "Could not load acceptance histogram from %s (%p)", fcfgAcceptance.value.c_str(), (void*)cfg.mAcceptance);
+      }
+    }
```

Comment:
I assume that something is not working at the moment on the CCDB side? Otherwise uncomment the L277